public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -368,6 +378,47 @@ export class LokiDatasource extends DataSourceApi<LokiQuery, LokiOptions> {
         return { status: 'error', message: message };
       });
   }
+
+  async annotationQuery(options: AnnotationQueryRequest<{ expr: string }>): Promise<AnnotationEvent[]> {
```

Comment:
Nitpick: A specific loki type LokiAnnotationQuery would help readability. Find dynamic type creations in generic args hard to read but not super important.

Reply:
Changed the type a bit see additional comments later.